src/redux/middleware/ws/on_message.js

```diff
@@ -65,10 +65,13 @@ export default (alias, store) => (e = {}) => {
       }
 
       case 'info.auth_token': {
-        const [, token] = payload
+        const [, token, hasCredentials] = payload
         store.dispatch(WSActions.recvAuthToken(token))
         store.dispatch(AOActions.getActiveAlgoOrders())
         store.dispatch(WSActions.send(['strategy.execute_status', token]))
+        if (hasCredentials) {
+          store.dispatch(WSActions.send(['get.order_history', token]))
+        }
```

Comment:
order history should be fetched after apiClientConfigured is true. otherwise following case won't work:
- user logs in app
- will receive auth_token but api credentials are not set up at this point
- use enters api keys
- at this point order history will be empty and so it should be refetched whenever api keys state changes 
`const apiClientConfigured = apiCredentials?.configured && apiCredentials?.valid`

Reply:
`hasCredentials` has the same meaning on the backend https://github.com/bitfinexcom/bfx-hf-server/blob/f93711be7055184ec97b13d9cf8b044b7de39da5/lib/ws_servers/api/send_authenticated.js#L49

I'll rename it so we have the same on both places

Reply:
no, but what I mean is info.auth won't be sent again when api keys are entered, so fetching order history call here won't work and it will reintroduce [this issue](https://app.asana.com/0/1201849173362898/1202361239080775/f) I fixed few weeks back